demosplan/DemosPlanCoreBundle/ResourceTypes/StatementVoteResourceType.php

```diff
@@ -70,17 +72,21 @@ protected function getProperties(): ResourceConfigBuilderInterface
         $statementVoteConfig->id->setReadableByPath();
 
         $statementVoteConfig->name
-            ->readable(true, fn (StatementVote $statementVote): string => $statementVote->getName())
-            ->updatable([$voteConditions], function (StatementVote $statementVote, ?string $name): array {
+            ->setReadableByCallable(fn (StatementVote $statementVote): string => $statementVote->getName())
+            ->addUpdateBehavior(new CallbackAttributeSetBehaviorFactory([], static function (StatementVote $statementVote, ?string $name): array {
```

Comment:
Why you removed the $voteConditions?

Reply:
@breierDemos good catch ! I added it now f0186d7b7523f5f7abe5b4b9c9d9895894e669de thanks for the feedback